packages/protocol/test-sol/unit/common/GoldToken.t.sol

```diff
@@ -55,16 +48,14 @@ contract CeloTokenTest_PreL2 is CeloTokenTest {
   function setUp() public {
     super.setUp();
 
-    vm.prank(address(0));
-    celoToken.mint(celoUnreleasedTreasuryAddress, L2_INITIAL_STASH_BALANCE);
     vm.deal(celoUnreleasedTreasuryAddress, L2_INITIAL_STASH_BALANCE);
 
     vm.deal(address(0), 0);
   }
 }
 contract CeloTokenTest_L2 is CeloTokenTest_PreL2, WhenL2 {}
```

Comment:
Since we only need L2 now, shouldn't all the L2-related setup just go straight into the CeloTokenTest main contract, and all other test suites just inherit from that?

Reply:
And could/should `WhenL2` just be added to the top-level Test contract so all tests by default are in that environment?